Dockerfile

```diff
@@ -12,19 +12,29 @@ RUN buildDeps=' \
         libencode-perl \
         make \
     ' \
+    url="https://rakudo.org/downloads/star/rakudo-star-${rakudo_version}.tar.gz" \
+    keyserver='hkps.pool.sks-keyservers.net' \
```

Comment:
As written, this won't actually use hkps (it'll use hkp against the hkps pool, which will generally work), but I'd also argue that when using a full fingerprint, hkps doesn't provide additional benefit since gpg will verify the fingerprint of the key after fetching which is a stronger guarantee, and not using hkps allows us to intentionally mitm for increased reliability (see docker-library/php#666), which we do on all the official build servers for the official images.

Reply:
Thanks for the insight! I'll update the branch to use the 'ha' pool as suggested on the official image readme.